02_Development/Display/Tests/test_discretetransferfunction.py

```diff
@@ -69,7 +68,7 @@ def testGetDatum_firstOrderTerm(self):
                                    "in the numerator")
 
     def testGetDatum_secondOrderTerm(self):
-        
+
```

Comment:
what's with the extra space but no comment in the space?  No need for space between function headers and function body